src/Microsoft.VisualStudio.Services.Agent/Constants.cs

```diff
@@ -228,5 +229,17 @@ public static class Task
                 public static readonly string DisplayName = "task.displayname";
             }
         }
+
+        public class Release
+        {
+            public class Path
+            {
+                public static readonly string ReleaseDirectoryPrefix = "r";
```

Comment:
Discussed with sriram, he also feels we should use r1... r2.. etc and not r/1..
I will update the PR with those changes and add UT's